basic_block_gp/blockchain.py

```diff
@@ -130,19 +136,51 @@ def valid_chain(self, chain):
 blockchain = Blockchain()
 
 
-@app.route('/mine', methods=['GET'])
+@app.route('/mine', methods=['POST'])
```

Comment:
For the basic blockchain exercise, changing the HTTP request for the mine endpoint to POST is not necessary.